### PR TITLE
fix(log): map SNAPCRAFT_VERBOSITY_LEVEL to CRAFT

### PR DESCRIPTION
**Pull request by syu-w**

### CODE REVIEW

**Review by tigarmo — APPROVED** [review]

File: requirements-devel.txt
Comment (mr-cal) [inline requirements-devel.txt:14]: Are you planning to wait for a new craft-application release before landing this?

**Review by mr-cal — APPROVED** [review]
Looks good!

File: requirements-devel.txt
Comment (syu-w) [reply requirements-devel.txt:14]: Yes

**Pull request merged**
